osconfig_tests/compute/instance.go

```diff
@@ -82,6 +88,52 @@ func (i *Instance) WaitForSerialOutput(match string, port int64, interval, timeo
 	}
 }
 
+// StreamSerialOutput stores the serial output of an instance to GCS bucket
+func (i *Instance) StreamSerialOutput(ctx context.Context, storageClient *storage.Client, logsPath, bucket string, wg *sync.WaitGroup, port int64, interval time.Duration) {
+	defer wg.Done()
```

Comment:
minor: consider renaming `wg` to `logwg`. That will make it easier to map these to the ones where you call `logwg.Add(1)` and `logwg.Wait()`.

Reply:
done